src/CodeReport/AnnotateDisassemblyTest.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/strings/str_replace.h>
+#include <gtest/gtest.h>
+
+#include <filesystem>
+
+#include "CodeReport/AnnotateDisassembly.h"
+#include "CodeReport/AnnotatingLine.h"
+#include "CodeReport/DisassemblyReport.h"
+#include "OrbitBase/ExecutablePath.h"
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/ReadFileToString.h"
+#include "capture_data.pb.h"
+
+using namespace std::string_view_literals;
+const std::string_view kMainFunctionInstructions =
+    "\x50\xbf\x04\x20\x40\x00\xe8\xe5\xfe\xff\xff\x31\xc0\x59\xc3\x90"sv;
+
+static void TestSimple(bool windows_line_endings) {
```

Comment:
Maybe this could be a more expressive name than `TestSimple`, but I also do not have a better idea and I do not care too much. Feel free to dismiss

Reply:
Yes. Good point. I added a comment.

Reply:
Yes, this is also frustrating to me because I don't have a better name in mind. So I leave it for now. There will probably be more tests in the future. Then it will be easier to give them differentiating names.